docs/hello_nextflow/01_hello_world.md

```diff
@@ -557,19 +557,19 @@ In principle we can write it anywhere; but since we're going to want to give it
 
 In the workflow block, make the following code change:
 
-_Before:_
+=== "After"
 
-```groovy title="hello-world.nf" linenums="24"
-// emit a greeting
-sayHello()
-```
+    ```groovy title="hello-world.nf" linenums="24" hl_lines="2"
```

Comment:
I'm also not sure about highlighting lines in cases like this, where it's basically the entire code block. I think this would be clearer without any highlight.

Reply:
I think it could be confusing that we always highlight the changes and here in this example we don't. It doesn't hurt to highlight the single line, even if it's a single line.